### PR TITLE
Updated sharma.md file

### PR DESCRIPTION
**Pull request by veriventiNarsimhaSharma**
#Done
Pull request merge the branch
#To do
delete branch later

### CODE REVIEW

File: _slides/07-veriventiNarsimhaSharma.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello Git
```
Comment (veriventiNarsimhaSharma) [inline _slides/07-veriventiNarsimhaSharma.md:6]: Looks Good

**Review by veriventiNarsimhaSharma — COMMENTED** [review]
good!

**Comment by veriventiNarsimhaSharma** [comment]
Looks well!

**Review by michaelca2024 — APPROVED** [review]
Looks good

**Pull request merged**